### PR TITLE
LPD-34740 Add new APIs for Console integration

### PR DESCRIPTION
**Pull request by kevenleone**

### CODE REVIEW

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-etc-spring-boot/src/main/java/com/liferay/marketplace/KoroneikiRestController.java [hunk 1]
```diff
@@ -72,29 +52,30 @@
...
 
-		Order order = _orderResource.getOrder(orderId);
+		Order order = _marketplaceService.getOrderResource(
+		).getOrder(
+			orderId
+		);
 
 		for (OrderItem orderItem :
-				_orderItemResource.getOrderIdOrderItemsPage(
+				_marketplaceService.getOrderItemResource(
```
Comment (nNilton) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-etc-spring-boot/src/main/java/com/liferay/marketplace/KoroneikiRestController.java:66]: Avoid getting the same resource during loop iterations

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-etc-spring-boot/src/main/java/com/liferay/marketplace/KoroneikiRestController.java [hunk 2]
```diff
@@ -72,29 +52,30 @@
...
 					com.liferay.headless.commerce.admin.order.client.pagination.
 						Pagination.of(1, 10)
 				).getItems()) {
 
 			ProductPurchaseView productPurchaseView =
-				_productPurchaseViewResource.
-					getAccountAccountKeyProductProductKeyProductPurchaseView(
-						order.getAccountExternalReferenceCode(),
-						orderItem.getSkuExternalReferenceCode());
+				_koroneikiService.getProductPurchaseViewResource(
```
Comment (nNilton) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-etc-spring-boot/src/main/java/com/liferay/marketplace/KoroneikiRestController.java:74]: Avoid getting the same resource during loop iterations

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-etc-spring-boot/src/main/java/com/liferay/marketplace/KoroneikiRestController.java [hunk 3]
```diff
@@ -229,9 +214,12 @@ public void postProduct(
 
 			String name = productName + " - " + dxpLicenseUsageType;
 
+			ProductResource productResource =
+				_koroneikiService.getProductResource();
```
Comment (nNilton) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-etc-spring-boot/src/main/java/com/liferay/marketplace/KoroneikiRestController.java:218]: Avoid getting the same resource during loop iterations

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**